src/services/WolfDen.Domain/Entity/LeaveType.cs

```diff
@@ -0,0 +1,42 @@
+﻿using WolfDen.Domain.Enum;
+
+namespace WolfDen.Domain.Entity
+{
+    public class LeaveType
+    {
+        public int Id {  get; private set; }
+        public string TypeName { get;private set; }
+        public int MaxDays { get;private set; }
+        public bool HalfDay { get; private set; }
```

Comment:
isHalfDayAllowed. better to rename this. because we are already using HalfDay in another entity for another purpose

Reply:
HalfDay changed to IsHalfDayAllowed